packages/react-components/react-spinbutton/src/components/SpinButton/useSpinButton.tsx

```diff
@@ -47,7 +47,7 @@ export const useSpinButton_unstable = (props: SpinButtonProps, ref: React.Ref<HT
   const nativeProps = getPartitionedNativeProps({
     props,
     primarySlotTagName: 'input',
-    excludedPropNames: ['onChange', 'size', 'min', 'max'],
+    excludedPropNames: ['onChange', 'size', 'min', 'max', 'value', 'defaultValue'],
```

Comment:
_nit:_ Order these alphabetically

```suggestion
    excludedPropNames: ['defaultValue', 'max', 'min', 'onChange', 'size', 'value'],
```